code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -1155,6 +1161,20 @@ static HookFunction initFunction([]()
 		}
 	});
 
+	fx::ScriptEngine::RegisterNativeHandler("GET_VEHICLE_HAS_FLAG", [](fx::ScriptContext& context)
+	{
+		int flagIndex = context.GetArgument<int>(1);
+
+		if (flagIndex < 0 || flagIndex >= (32 * 7)) return;
```

Comment:
We prefer to use brackets even for single-line returns like that. It should be something like this:
```cpp
if (flagIndex < 0 || flagIndex >= (32 * 7))
{
    return;
}
```

While I assume it's ~okay to keep a hardcoded max index flag for now (we're far from any out-of-bounds issues as of now), how difficult do you think it would be for you to get the number of flags directly from rage's parser? I could implement this myself later if you feel uncomfortable about changing the way it works in this PR.